platformio.ini

```diff
@@ -12,6 +12,7 @@ lib_deps =
     ;https://github.com/h2zero/NimBLE-Arduino.git#master //Nimble is tracked and including this will probably cause issues. 
     teemuatlut/TMCStepper@^0.7.1
     bblanchon/ArduinoJson@^6.17.2
+    ESP Async WebServer
```

Comment:
It is a good practice to pin to a version, if possible, so that we builds can be idempotent.